### PR TITLE
example: changed to use the new config style for downstream_protocol & upstream_protocol.

### PR DESCRIPTION
**Pull request by doujiang24**

### CODE REVIEW

File: examples/codes/dubbo/client_config.json [hunk 1]
```diff
@@ -45,12 +45,8 @@
                 {
                   "type": "proxy",
                   "config": {
-                    "downstream_protocol": "X",
-                    "upstream_protocol": "X",
```
Comment (antJack) [inline examples/codes/dubbo/client_config.json:49]: "upstream_protocol" is no more needed?

Comment (doujiang24) [reply examples/codes/dubbo/client_config.json:49]: yeah, it defaults to `downstream_protocol` now.

**Review by taoyuanyuan — APPROVED** [review]
LGTM

**Review by antJack — APPROVED** [review]
LGTM

**Pull request merged**